### DISCUSSION TITLE
关于博士论文的图表标号方式

### DISCUSSION THREAD

**Discussion by chuyan99** [sjtug/SJTUThesis#1063] [提问]
请问模版运行出来的图标是1.1，怎么修改为1-1。

**Comment by LogCreative** [comment]
新版[参考模板](https://www.gs.sjtu.edu.cn/post/detail/Z3M2MjU=)使用句点`.`分隔章节号和标号，如果想使用 `-` 分隔，请使用：
```latex
\sjtusetup{style/num-sep={-}}
```
也可以逐项更改 `float-num-sep`、`equation-num-sep`、`theorem-num-sep`。

相关：#1043
